osh.spec

```diff
@@ -31,6 +32,7 @@ BuildRequires:  systemd-rpm-macros
 %description
 OpenScanHub is a service for static and dynamic analysis.
 It consists of central hub, workers and cli client.
+It runs various static (and dynamic) analyzers on RPM packages.
```

Comment:
I find the description a bit clumsy.  The first and last sentence duplicate each other.  Cannot you just append ` of RPM packages` to the first sentence?